spec/index.bs

```diff
@@ -384,14 +384,14 @@ interface TrustedHTML {
 };
 </pre>
 
-TrustedHTML objects have a <dfn export for="TrustedHTML">\[[Data]]</dfn> internal slot which holds a
+TrustedHTML objects have a <dfn export for="TrustedHTML">Data</dfn> internal slot which holds a
```

Comment:
"data", not "Data".

And it's also not an "internal slot". I'd use the word associated as we have done elsewhere.

You also want to switch DOMString for string. We hardly use IDL types in specification text.